### ISSUE TITLE
ajax not working after activate plugin

### ISSUE THREAD

**Issue by inblues** [newtribesmission/NTM-WPMem-Email-As-Username#6]
Ajax call will not work if activate the plugin. Error message captured: 'AJAX ERROR: no response'.  // probably security issue?

 jQuery.post( 
        wp_ajax.ajaxurl, 
        data,  
        function( response )
        {
            // Errors
            if( !response.success )
            {
                // No data came back, maybe a security error
                if( !response.data )
                {
                    console.log( 'AJAX ERROR: no response' );

```
            }
            // Error from PHP
            else
            {
                console.log( 'Response Error: ' + response.data.error );

            }
        }
        // Success
        else{
          console.log( 'Response Success: ' + response.data );
            jQuery("#myresult").html('');
            jQuery("#myresult").append(response.data);

        }
    }
); 
```

**Comment by stevish-com** [comment]
Hmm, I can't seem to find anything that would be interfering with ajax or javascript at all. Where is this ajax code of yours being implemented?

**Comment by inblues** [comment]
Yes, Thanks for quite response.  it's in http://fengshuizixin.com/%E5%A7%93%E5%90%8D%E5%88%86%E6%9E%90/ . If i activate the plugin, the chinese name analysis won't work. If i deactivate the plugin, it work perfectly. Now it's working fine after deactivated the plugin. If you wish to do some testing, please let me know. I can turn on/off the plugin.

**Comment by stevish-com** [comment]
Would you mind posting the complete code from your plugin that's making the ajax call? I can't seem to find anything in my code that would collide with an ajax call.

**Comment by inblues** [comment]
Thanks you very much. I have my code ready, how to send to you?

**Comment by inblues** [comment]
I have tested with firefox and IE which is not working but no problem with google chrome. Please download the plugin here https://drive.google.com/file/d/0B5RtMIWpZ-0Mbk93WVAyeWNfdGM/view?usp=sharing 

Thank you very much
